src/main/webapp/account.jsp

```diff
@@ -4,6 +4,8 @@
     <meta charset="UTF-8">
     <title>My Personal Info</title>
     <link rel="stylesheet" href="account_style.css">
+    <script src="https://maps.googleapis.com/maps/api/js?key=YOUR_API_KEY_HERE&libraries=places&language=en;"></script>
```

Comment:
Just for consistency, I would recommend using the config.js file to store the API key, like we did for the Admin and Homepage, that way all of our files are using the same method to retrieve the API key.

Reply:
I tried to use the config.js file to store the API key but the Geocoding API always fails if I didn't include the key in the jsp file, which is really weird.

Reply:
That's odd, did you include this before your maps API script tag? `<script type='text/javascript' src='config.js'></script>`

Reply:
I did, but the Geocoding services didn't give me any response.

Reply:
Ah, I think including a JS variable doesn't work in HTML/JSP, so you'd have to dynamically load the script tag within your initMap() function. I'd try adding the following at the beginning of your initMap() function:

const script = document.createElement("script");
script.type = "text/javascript";
script.src =  "https://maps.googleapis.com/maps/api/js?key=" + MAPSKEY + "&callback=init&libraries=places&language=en";
script.defer = true;
script.async = true;
document.head.appendChild(script);

window.init = async function() {
   --your current initMap() function would go inside here--
}

Reply:
Cool! I would try it out tomorrow. 👍

Reply:
+1 - let's get all the maps working using the same key-loading mechanism

Reply:
I just made a new commit and switched to the same key-loading mechanism as Mafe and Denzil.